pkg/os/windows/powershell/powershell_windows.go

```diff
@@ -4,46 +4,17 @@ import (
 	"bytes"
 	"io/ioutil"
 	"os"
+	"os/exec"
 	"path/filepath"
 	"strings"
 	"syscall"
 
-	"os/exec"
-
 	"github.com/code-ready/crc/pkg/crc/logging"
 )
 
-var (
-	runAsCmds = []string{
-		`$myWindowsID = [System.Security.Principal.WindowsIdentity]::GetCurrent();`,
-		`$myWindowsPrincipal = New-Object System.Security.Principal.WindowsPrincipal($myWindowsID);`,
-		`$adminRole = [System.Security.Principal.WindowsBuiltInRole]::Administrator;`,
-		`if (-Not ($myWindowsPrincipal.IsInRole($adminRole))) {`,
-		`  $procInfo = New-Object System.Diagnostics.ProcessStartInfo;`,
-		`  $procInfo.FileName = "` + LocatePowerShell() + `"`,
-		`  $procInfo.WindowStyle = [Diagnostics.ProcessWindowStyle]::Hidden`,
-		`  $procInfo.Arguments = "-ExecutionPolicy RemoteSigned & '" + $script:MyInvocation.MyCommand.Path + "'"`,
-		`  $procInfo.Verb = "runas";`,
-		`  [System.Diagnostics.Process]::Start($procInfo);`,
-		`  Exit;`,
-		`}`,
-	}
-	isAdminCmds = []string{
-		"$currentPrincipal = New-Object Security.Principal.WindowsPrincipal([Security.Principal.WindowsIdentity]::GetCurrent())",
-		"$currentPrincipal.IsInRole([Security.Principal.WindowsBuiltInRole]::Administrator)",
-	}
-
-	poshStdLocation = filepath.Join(os.Getenv("SYSTEMROOT"), "System32", "WindowsPowerShell", "v1.0", "powershell.exe")
-)
-
-func LocatePowerShell() string {
```

Comment:
I think this was only needed for windows 7 and got here from minishift, (or if the $PATH has been messed with), safe to remove now, since we don't support win7